.gitignore

```diff
@@ -67,5 +67,4 @@ app/build.gradle
 app/src/main/res/values/api_keys.xml
 app/src/main/res/values/test_accounts.xml
 app/src/main/res/values/secrets.xml
-app/src/main/java/com/example/bookwormadventuresdeluxe2/Utilities/NotificationUtility/CloudMessagingServerKeyConstant.java
```

Comment:
Leave this one in the gitignore as it is the new path, remove the one below. Remove the other file from github pls